mne/decoding/tests/test_time_gen.py

```diff
@@ -424,4 +424,23 @@ def test_decoding_time():
     tg = TimeDecoding(clf=clf, cv=cv)
     tg.fit(epochs, y=y)
 
+
+@requires_sklearn
+def test_string_scorer():
+    """Test whether passing string as scorer works as intended"""
+    epochs = make_epochs()
+    gat_1 = GeneralizationAcrossTime(scorer='accuracy')
```

Comment:
add `epochs.crop(epochs.times[0], epochs.times[2])` here to limit the duration of this test.

Also, change these `GeneralizationAcrossTime` objects with `TimeDecoding` objects, it will run a bit faster.